source/index.js

```diff
@@ -1,5 +1,13 @@
-export {getStreamAsArray} from './array.js';
-export {getStreamAsArrayBuffer} from './array-buffer.js';
-export {getStreamAsBuffer} from './buffer.js';
-export {getStreamAsString as default} from './string.js';
-export {MaxBufferError} from './contents.js';
+import {on} from 'node:events';
+import {finished} from 'node:stream/promises';
+import {nodeImports} from './stream.js';
+
+Object.assign(nodeImports, {on, finished});
```

Comment:
An alternative would be to bind those with each exported method.
However, using a top-level variable results in simpler code, since those "Node imports" do not need to be passed around through the codebase.